src/DynamoCore/Graph/Workspaces/WorkspaceInfo.cs

```diff
@@ -13,6 +13,19 @@ namespace Dynamo.Graph.Workspaces
     /// </summary>
     public class WorkspaceInfo
     {
+        public WorkspaceInfo(string id, string name, string description, RunType runType)
```

Comment:
API Change : A new public constructor was added for `WorkspaceInfo` to facilitate deserialization.